presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
```

Comment:
Since the ldap config variables are only used by the ldap tests, it doesn't do any harm to have them in the standard config, and it gives us one less (mostly identical) config file to maintain.

If we could merge tempto config files at runtime, I would much prefer to do that.